elasticsearch-extensions/lib/elasticsearch/extensions/test/cluster.rb

```diff
@@ -102,8 +101,11 @@ def start(arguments={})
             return false
           end
 
+          # Flag to clear the cluster. May be passed directly with args or via ENV variables.
+          clear_cluster = !!arguments[:clear] || (ENV.fetch('TEST_CLUSTER_CLEAR', 'true') != 'false')
```

Comment:
Why not put the `ENV.fetch ...` into the arguments block above as with the rest of the options? It would make it more consistent I think?